### PR TITLE
Addshore rewrite

### PR DESCRIPTION
**Pull request by addshore**

### CODE REVIEW

File: java/analyzer/src/main/java/org/wikidata/analyzer/DumpFile/JsonLocalDumpFileImpl.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import org.wikidata.wdtk.util.DirectoryManager;
+import org.wikidata.wdtk.util.DirectoryManagerImpl;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Implements the unimplemented class JsonLocalDumpFile from the ToolKit
+ * This class is removed in a future version and this this may be rethought
```
Comment (Benestar) [inline java/analyzer/src/main/java/org/wikidata/analyzer/DumpFile/JsonLocalDumpFileImpl.java]: -this

Comment (addshore) [inline java/analyzer/src/main/java/org/wikidata/analyzer/DumpFile/JsonLocalDumpFileImpl.java]: Fixing in the next commit

File: python/geo2png.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
+# trafo -> png
+# Original source https://github.com/vrandezo/wikidata-analytics/blob/6896d639bd124eec7571f1179f27ae2b6bfaa435/geo2png.py
```
Comment (JanZerebecki) [inline python/geo2png.py]: Source has no licence. Does this copy need more attribution?

File: README.md [hunk 3]
```diff
@@ -1,14 +1,6 @@
...
 
-
-#### to run in other environments
-To run in other environments just run run.sh
-
-* NOTE: the scripts require a lot of memory
-* NOTE: the scripts expect public_html to be in your home dir
-* NOTE: analysis.sh expects to be run when in its path
\ No newline at end of file
+Please run your Java VM with -Xmx2g or greater.
```
Comment (JanZerebecki) [inline README.md]: Can you document how to run this?

File: python/geo2png.py (hunk 2 above)
Comment (addshore) [inline python/geo2png.py]: All I have to go on is this
https://github.com/wmde/wikidata-analysis/blob/addshoreRewrite/html/map/index.html#L123
The original tool says that all code is PD

File: README.md (hunk 3 above)
Comment (addshore) [inline README.md]: This readme is out of date / in the wrong place:
See https://github.com/wmde/wikidata-analysis/blob/addshoreRewrite/java/analyzer/README.md
I have removed the line above (in the next commit)

File: html/map/index.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+A map of Berlin showing the
+<a href="https://www.wikidata.org/wiki/Property:P197"><i>adjacent station</i></a>
+property, i.e. the public transport stations and how they are connected.
+				</p>
+				<footer>
+					<p>
+The Wikidata map was created 2013 by
+<a href="http://denny.vrandecic.de">Denny Vrandecic</a>.
+The map is now maintained by WMDE.
+All data, images, and code are in the public domain.
```
Comment (JanZerebecki) [inline html/map/index.html]: This is contradictory with the LICENSE file.

File: LICENSE [hunk 5]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2013 Denny Vrandecic
```
Comment (JanZerebecki) [inline LICENSE]: Is there nothing left that might have been copied from Denny?

File: python/geo2png.py (hunk 2 above)
Comment (JanZerebecki) [inline python/geo2png.py]: That is from your additions not from where you copied that code from. So we need @vrandezo to license it.

Comment (addshore) [inline python/geo2png.py]: Well, the original is at http://tools.wmflabs.org/wikidata-analysis/pre2015/map/ (not added by me)
I am fine with waiting for @vrandezo though...

Comment (vrandezo) [inline python/geo2png.py]: Yes, I confirm, my original code was published into the Public Domain, or, if that leaves any legal doubt, under CC0.

File: html/map/index.html (hunk 4 above)
Comment (addshore) [inline html/map/index.html]: LICENSE file has been updated

**Pull request merged**